src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -165,7 +161,7 @@ private function getMetadata($type, $entity)
                     return false;
                 }
 
-                return !$param->getClass() || $param->getClass()->getName() !== 'Twig_Environment';
+                return !$param->getClass() || !in_array($param->getClass()->getName(), array('Twig_Environment', 'Twig\Twig'));
```

Comment:
@nicolas-grekas please revert this change to avoid useless conflicts with my PR removing this code entirely 😄